contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -793,7 +793,7 @@ exec_stmt_exec(PLtsql_execstate *estate, PLtsql_stmt_exec *stmt)
 	volatile LocalTransactionId before_lxid;
 	LocalTransactionId after_lxid;
 	volatile int rc;
-	SimpleEcontextStackEntry *topEntry;
+	SimpleEcontextStackEntry *volatile topEntry;
```

Comment:
why are we making all entries as volatile? are they all read/write across TRY/CATCH block